### PR TITLE
test: add filter to event feed

### PR DESCRIPTION
**Pull request by zzmp**
Adds a filter to the cosmos event feed to aid debugging.

### CODE REVIEW

File: src/cosmos/EventFeed.tsx [hunk 1]
```diff
@@ -66,15 +70,32 @@ export default function EventFeed({ events, onClear }: EventFeedProps) {
           <Type.Subhead1>clear</Type.Subhead1>
         </button>
       </Row>
+      <Row>
+        <Type.Subhead2>Filter: </Type.Subhead2>
+        <select
+          onChange={(e) => {
+            console.log(e.target.value)
```
Comment (lynnshaoyu) [inline src/cosmos/EventFeed.tsx:77]: is this console.log needed?

File: src/cosmos/EventFeed.tsx [hunk 2]
```diff
@@ -66,15 +70,32 @@ export default function EventFeed({ events, onClear }: EventFeedProps) {
...
+            onSelectEventType(e.target.value)
+          }}
+          value={selectedEventType}
+        >
+          <option>{SHOW_ALL_EVENTS}</option>
+          {HANDLERS.map((name) => (
+            <option key={name}>{name}</option>
```
Comment (lynnshaoyu) [inline src/cosmos/EventFeed.tsx:84]: any chance the handlers have the same name? could also use i (index) as key value for consistency with what you do on line 91

**Review by lynnshaoyu — COMMENTED** [review]
some comments, not sure if they should be blocking or not so didnt approve yet🙃

File: src/cosmos/EventFeed.tsx (hunk 2 above)
Comment (zzmp) [reply src/cosmos/EventFeed.tsx:79]: leaving this as a runtime check - left a comment stating as such

**Review by lynnshaoyu — APPROVED** [review]

**Pull request merged**